src/game/ChatCommands/MMapCommands.cpp

```diff
@@ -155,7 +155,7 @@ bool ChatHandler::HandleMmapLocCommand(char* /*args*/)
     int32 gx = 32 - player->GetPositionX() / SIZE_OF_GRIDS;
     int32 gy = 32 - player->GetPositionY() / SIZE_OF_GRIDS;
 
-    PSendSysMessage("%03u%02i%02i.mmtile", player->GetMapId(), gy, gx);
+    PSendSysMessage("%04u%02i%02i.mmtile", player->GetMapId(), gy, gx);
```

Comment:
this chage causes egmentation fault due to the format on this case is out of bounds